.github/workflows/static-analysis.yml

```diff
@@ -59,4 +59,6 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Psalm
-        uses: docker://vimeo/psalm-github-actions:3.17.2
+        uses: docker://vimeo/psalm-github-actions:4.1.1
+        env:
+          REQUIRE_DEV: "true"
```

Comment:
What is this flag for?

Reply:
See https://github.com/psalm/psalm-github-actions/pull/17